src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3380,6 +3380,12 @@ namespace winrt::TerminalApp::implementation
     // - Displays a info popup guiding the user into setting their default terminal.
     void TerminalPage::ShowSetAsDefaultInfoBar() const
     {
+        if (::winrt::Windows::UI::Xaml::Application::Current().try_as<::winrt::TerminalApp::App>() == nullptr)
+        {
+            // Just ignore this in the tests (where the Application::Current()
+            // is not a TerminalApp::App)
+            return;
+        }
```

Comment:
That feels like a rather... unique solution. 😄
Is there some other way to achieve this?

Reply:
Meh, there's actually a decent amount of precedent for doing this elsewhere in the app. It's not super semantically clean, but it does _work_ quite well.